rules/otbn.bzl

```diff
@@ -260,3 +300,20 @@ otbn_sim_test = rv_rule(
     toolchains = ["@rules_cc//cc:toolchain_type"],
     incompatible_use_toolchain_transition = True,
 )
+
+otbn_consttime_test = rule(
+    implementation = _otbn_consttime_test,
+    test = True,
+    attrs = {
+        "srcs": attr.label_list(allow_files = True),
+        "deps": attr.label_list(providers = [OutputGroupInfo]),
+        "subroutine": attr.string(),
+        "secrets": attr.string_list(),
+        "_checker": attr.label(
+            default = "//hw/ip/otbn/util:check_const_time.py",
+            executable = True,
+            cfg = "exec",
+            allow_files = True,
```

Comment:
I don't think you need `allow_files` when the target is a proper bazel label.  You _do_ need `allow_files` when the target is a file and not a bazel rule (e.g. if this were a script on the filesystem rather than a label to another rule known to bazel).